spec/features/user_adds_video_to_queue_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+
+feature 'user interacts with the queue' do
+  
+  scenario 'user adds videos to and reorders queue' do
+
+    videos = []
+    category  = Fabricate(:category)
+    videos[0] = Fabricate(:video, category: category)
```

Comment:
Giving the videos a name should be better for memory too. Like variable name should be enough.